packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -89,7 +89,7 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
     override lastCwd = new URI();
 
     @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService;
-    @inject(RemoteConnectionProvider) protected readonly conectionProvider: ServiceConnectionProvider;
+    @inject(RemoteConnectionProvider) protected readonly connectionProvider: ServiceConnectionProvider;
```

Comment:
Good catch! I wonder, since this field is not private, if this change could e.g. break extenders that might be accessing it using the typo-ed name in a class extending `TerminalWidgetImpl`? If so it could be better to leave it as is for how, and e.g. do a PR for it, that would be documented as a breaking change?

Reply:
It shouldn't be an issue as it is not yet part of any release (merged 3 days ago):

https://github.com/eclipse-theia/theia/commit/d2bc79bc1aac9cc424da7ac7945fbbb5d71f28c1#diff-da3ea75041150eb2403247c5ed84542210774c326c0e3b334029f4e3afb51fc0R92